demo/src/configs/radarConfigDefault.json

```diff
@@ -43,13 +43,13 @@
         "noiseWindow": 8,
         "guardLength": 4,
         "divideShift": 4,
-        "thresholdScale": 3072
+        "thresholdScale": 4096
     },
     "cfarCfgDoppler": {
         "noiseWindow": 4,
         "guardLength": 2,
         "divideShift": 3,
-        "thresholdScale": 3072
+        "thresholdScale": 4096
```

Comment:
I changed these to be 9 dB, because at 12 dB and hot it seemed too aggressive, so we may want to leave it there.